rosbag2_transport/test/rosbag2_transport/test_rewrite.cpp

```diff
@@ -75,11 +76,10 @@ class TestRewrite : public Test, public WithParamInterface<std::string>
 
   ~TestRewrite()
   {
-    // rcpputils::fs::remove_all(output_dir_);
```

Comment:
I think we need `fs::remove_all(output_dir)` here, since we creating a temp dir in the constructor.  
It seems It was commented out by mistake.